rpcs3/Emu/Cell/Modules/sceNp.cpp

```diff
@@ -3957,8 +3957,9 @@ error_code sceNpManagerGetAccountRegion(vm::ptr<SceNpCountryCode> countryCode, v
 	}
 
 	memset(countryCode.get_ptr(), 0, sizeof(SceNpCountryCode));
-	countryCode->data[0] = 'u';
-	countryCode->data[1] = 's';
+	static auto s_country_code = nph.get_country_code();
```

Comment:
Lol don't use static. This makes no sense at all.

```suggestion
	const auto s_country_code = nph.get_country_code();
```